modules/ethereum/src/lib.rs

```diff
@@ -370,6 +442,38 @@ impl<T: Trait> Module<T> {
 	}
 }
 
+impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
+	type Call = Call<T>;
+
+	fn validate_unsigned(call: &Self::Call) -> TransactionValidity {
+		match *call {
+			Self::Call::unsigned_import_header(ref header, ref receipts) => {
+				let accept_result = verification::accept_aura_header_into_pool(
+					&BridgeStorage::<T>::new(),
+					&kovan_aura_config(),
+					&kovan_validators_config(),
+					&pool_configuration(),
+					header,
+					receipts.as_ref(),
+				);
+
+				match accept_result {
+					Ok((requires, provides)) => Ok(ValidTransaction {
+						priority: TransactionPriority::max_value(),
+						requires,
+						provides,
+						longevity: TransactionLongevity::max_value(),
```

Comment:
Let's make it configurable from outside. I think it should have some more sane validity that depends on the block times of both blockchains.

Reply:
See top level comment - let's extract everything about configuration to separate issue.